.github/CODEOWNERS

```diff
@@ -1,6 +1,14 @@
 # Docs
 docs/**/*                              @iHiD @ErikSchierboom
 
+# Bot-generated
+languages/languages.json               @iHiD @ErikSchierboom
```

Comment:
@iHiD Let me know if you'd prefer not to be in this list.

Reply:
I'd probably rather not be :)

Reply:
Updated